### DISCUSSION TITLE
自己可以缩小gpu镜像吗

### DISCUSSION THREAD

**Discussion by yurinapoleon** [PaddlePaddle/PaddleOCR#12279] [Q&A]
CUDA12.3的镜像，解压完26.8个G，部署完直接上30，有什么办法能缩小点吗，太大了实在>_<，谢谢！

**Comment by tink2123** [comment]
官方镜像包含的内容较多，很抱歉暂时不支持缩小呢。

**Comment by TingquanGao** [comment]
建议用paddle官方镜像：https://www.paddlepaddle.org.cn/
